packages/mds-jurisdiction/handlers/create-jurisdiction.ts

```diff
@@ -22,17 +22,23 @@ import {
 import { isServiceError } from '@mds-core/mds-service-helpers'
 import { JurisdictionApiRequest, JurisdictionApiResponse } from '../@types'
 
+export type JurisdictionApiCreateJurisdictionRequestBody =
+  | CreateJurisdictionDomainModel
+  | CreateJurisdictionDomainModel[]
+
 export type JurisdictionApiCreateJurisdictionRequest = JurisdictionApiRequest<
-  CreateJurisdictionDomainModel | CreateJurisdictionDomainModel[]
+  JurisdictionApiCreateJurisdictionRequestBody
 >
```

Comment:
You could minimize the number of changes by altering the way these types are declared. For instance:
```ts
export type JurisdictionApiCreateJurisdictionRequest = JurisdictionApiRequest<
  CreateJurisdictionDomainModel | CreateJurisdictionDomainModel[]
>

export type JurisdictionApiCreateJurisdictionRequestBody = JurisdictionApiCreateJurisdictionRequest['body']
```
Additionally, you _could_ just use `JurisdictionApiCreateJurisdictionRequest['body']` and not declare the additional type at all although I am certainly not opposed to having a separate named type.

Reply:
It's coming clearer to me.  On that note I think I can just use `JurisdictionApiCreateJurisdictionRequest['body']` on the client side and just need to export what we have already.  What took me down this path was seeing that geography split things up like this

Reply:
Okay I've got the request types trimmed down.  It took me a while to realize that the ApiResponse is a different animal and doesn't have a body in the same way and isn't used on the client-side in the same way.